eng/emitters/pipelines/templates/steps/test-step.yml

```diff
@@ -48,6 +47,12 @@ steps:
     artifact: ${{ parameters.BuildArtifactName }}
     displayName: Download build artifacts
 
+  - task: NodeTool@0
+    displayName: Install Node.js
+    retryCountOnTaskFailure: 3
+    inputs:
+      versionSpec: ${{ parameters.NodeVersion }}
```

Comment:
Without this, the NodeVersion on test seems not having effect at all.

And we now need this step, as the default npm is [10.8.0](https://www.npmjs.com/package/npm/v/10.8.0), which causing problem with "nodejs>=20"

![image](https://github.com/user-attachments/assets/b85a55b9-059c-46c3-8232-0c8c3e2d36d4)

Reply:
i think npm is fine but it is still using node 18